WtWSMS/localization/english/bookmark/bm_476_l_english.yml

```diff
@@ -17,8 +17,8 @@
  bookmark_476_theodoric:0 "Theodoric"
  bookmark_476_theodoric_desc:0 "Theodoric, raised a hostage of the Eastern Roman Empire, was raised a Roman and received a complete education. In 473, he would become the leader of the Ostrogoths of Pannonia, and less than ten years later, unite the Ostrogoths under his rule. However, his greatest achievement would be to overthrow Odovacar and establish an Ostrogothic Kingdom across the entire Italian Peninsula. Vying for a return to greatness, like that under the ancient Roman Empire, will his dreams come to fruition?"
  
- bookmark_476_julius_nepos:0 "Julius"
- bookmark_476_julius_nepos_desc:0 "Julius Nepos, seen by some as the last legitimate Western Roman Emperor, was its de facto ruler following the ascension of Romulus Augustus, and then the barbarian Odovacar. In control of a mere fragment of his empire, a small holdout in Dalmatia, Julius gears up for a massive invasion to reclaim his rightful land - starting with the Italian Peninsula.\n\n#bold Will the true claimant succeed?#!"
+ bookmark_476_julius_nepos: "Nepos"
```

Comment:
While Julius is a generic name for Romans, it is a bit weird to have everyone adressed by their given name and then have "Nepos"

Reply:
> While Julius is a generic name for Romans, it is a bit weird to have everyone adressed by their given name and then have "Nepos"

He is usually shortly referred by his cognomen, i. e Nepos. This change was made to reflect the addition of a character, whose name in the game is Nepos Metilius (Publius Metilius Nepos).

Reply:
Ok, but the other Nepos isn't accessible as a special interest character via bookmark